pkg/tsdb/stackdriver/stackdriver.go

```diff
@@ -411,7 +435,7 @@ func (e *StackdriverExecutor) parseResponse(queryRes *tsdb.QueryResult, data Sta
 				points = append(points, tsdb.NewTimePoint(null.FloatFrom(value), float64((point.Interval.EndTime).Unix())*1000))
 			}
 
-			metricName := formatLegendKeys(series.Metric.Type, defaultMetricName, series.Resource.Type, series.Metric.Labels, series.Resource.Labels, make(map[string]string), query)
+			metricName := formatLegendKeys(series.Metric.Type, defaultMetricName, seriesLabels, make(map[string]string), query)
```

Comment:
Instead of passing an empty map, you could just pass `nil`. `formatLegendKeys` does not mutate the map, reading from a nil map will not produce a panic.

Reply:
Yes, thank you!